### PR TITLE
Add Nobl9 tile to integration-extras

### PR DESCRIPTION
**Pull request by ian-bartholomew**
### What does this PR do?

Add a [Nobl9](https://www.nobl9.com) tile

### Motivation

To promote Nobl9's integration with Datadog

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?
Nobl9 works by querying Datadog, so there isn't any integration needed on the Datadog side.

### CODE REVIEW

File: nobl9/README.md
Comment (cswatt) [inline nobl9/README.md:43]:
```suggestion
**Name** field.
```

File: nobl9/README.md
Comment (cswatt) [inline nobl9/README.md:44]:
```suggestion
4. Enter a name for your data source (required). Names are required because metadata names are
```

File: nobl9/README.md
Comment (cswatt) [inline nobl9/README.md:48]:
```suggestion
5. Enter a description (optional). Add the team or owner details and explain
```

File: nobl9/README.md
Comment (cswatt) [inline nobl9/README.md:57]:
```suggestion
See Nobl9's [User Guide][1] for more information about creating SLOs on the Nobl9 platform.

## Troubleshooting

Need help? Contact [Nobl9 Support][2] or [Datadog Support][3].

[1]: https://nobl9.github.io/techdocs_User_Guide/#service-level-objectives-38
[2]: https://nobl9.com/about/#contact
[3]: https://docs.datadoghq.com/help/

```

File: nobl9/README.md
Comment (wimsikal) [reply nobl9/README.md:19]: Use the Datadog Nobl9 integration to pass business context to monitoring data, define and measure reliability goals, and align activities against the priorities set by the error budget.

File: nobl9/README.md
Comment (wimsikal) [reply nobl9/README.md:38]: `https://api.datadoghq.com` or `https://api.datadoghq.eu`

**Comment by yzhan289** [comment]
CI is failing because:
```
Integration nobl9 has a `CODEOWNERS` entry, but the codeowner is not a username or team.
```

Can you fix the CODEOWNERS file to use a Github username? Something like

```
/nobl9/                                  @ian-bartholomew support@nobl9.com
```

File: nobl9/manifest.json
Comment (yzhan289) [inline nobl9/manifest.json:4]: I believe the `manifest_version` for `extras` integrations should be `2.0.0` now? Is that correct @mgarabed?

Comment (mgarabed) [reply nobl9/manifest.json:4]: Nice enthusiasm - but let's hold off on transitioning / migrating other integrations just yet.  We can sync after DASH about what a good approach should be going forward.

**Comment by wimsikal** [comment]
@cswatt @yzhan289 - The PR is ready for review.

File: nobl9/README.md
Comment (sarah-witt) [inline nobl9/README.md:5]: Can you list these metrics in the `metadata.csv`?

File: nobl9/manifest.json
Comment (sarah-witt) [inline nobl9/manifest.json:31]:
```suggestion
```
You don't need a EULA

File: nobl9/manifest.json
Comment (sarah-witt) [inline nobl9/manifest.json:19]:
```suggestion
```

File: nobl9/manifest.json
Comment (sarah-witt) [inline nobl9/manifest.json:35]:
```suggestion
```

File: nobl9/manifest.json
Comment (sarah-witt) [inline nobl9/manifest.json:7]: If metrics are emitted can you add one here?

**Review by sarah-witt — CHANGES_REQUESTED** [review]
Hi, thank you for the contribution! Sorry for the delay- I left a few comments!

File: nobl9/README.md
Comment (ian-bartholomew) [reply nobl9/README.md:5]: @sarah-witt per @coignetp 's comment [here](https://github.com/DataDog/integrations-extras/pull/1019#discussion_r736232740) it sounds like its not needed.

**Comment by ian-bartholomew** [comment]
@coignetp @sarah-witt updated the PR per your feedback.  Thanks!

**Review by ChristineTChen — APPROVED** [review]

**Review by sarah-witt — APPROVED** [review]

File: nobl9/README.md
Comment (alai97) [inline nobl9/README.md:8]:
```suggestion
Nobl9 is a SLO platform that provides real-time, historical SLO reports. Nobl9 integrates with Datadog to collect SLI metrics and measure them against SLO targets. Since Nobl9 calculates error budgets of acceptable thresholds, it can trigger workflows and alerts when the error burn rate is too high or has been exceeded.
```

File: nobl9/README.md
Comment (alai97) [inline nobl9/README.md:12]:
```suggestion
The Datadog Nobl9 integration allows you to pass business context through monitoring data, define and measure reliability goals, and align activities against priorities set by the error budget.
```

File: nobl9/README.md
Comment (alai97) [inline nobl9/README.md:27]:
```suggestion

All configuration happens on the Nobl9 SLO Platform.
```

File: nobl9/README.md
Comment (alai97) [inline nobl9/README.md:42]:
```suggestion
1. Add the Datadog API endpoint to connect to your data source, either `https://api.datadoghq.com/` or `https://api.datadoghq.eu/` (required).
2. Enter a **Project** name. This field is intended for situations where multiple users are spread across multiple teams or projects. When the field is left blank, a default value appears.
3. The **Display Name** appears automatically when a name is entered in the **Name** field.
4. Enter a name for your data source (required). Metadata names are unique within each project and are validated against some RFC and DNS names. The data source name must contain only lowercase alphanumeric characters and dashes. For example: `my-datadog-data-source`.
5. Enter a description (optional). Add the team or owner details and explain why you created this specific data source. Descriptions provide immediate context for any team member.
```

File: nobl9/README.md
Comment (alai97) [inline nobl9/README.md:44]:
```suggestion
For more information about creating SLOs on the Nobl9 platform, see Nobl9's [User Guide][1].
```

File: nobl9/README.md
Comment (alai97) [inline nobl9/README.md:59]: Looks like a duplicate here: 

```suggestion
```

**Review by alai97 — CHANGES_REQUESTED** [review]

**Comment by ChristineTChen** [comment]
@ian-bartholomew Please see the above documentation review. Once that's addressed, this PR is good to go.

**Comment by ian-bartholomew** [comment]
Thanks @ChristineTChen ! I added the changes.

**Review by alai97 — APPROVED** [review]
Looks good for docs!

**Comment by ChristineTChen** [comment]
Hey @ian-bartholomew 

The Nobl9 tile is now available in the Integrations page

**Comment by ian-bartholomew** [comment]
@ChristineTChen Great! Thank you

**Pull request merged**
